src/styles/header.scss

```diff
@@ -0,0 +1,75 @@
+:root {
+  --primary-color: #00acdc;
+}
```

Comment:
It's a good practice to keep all color definitions in a separate file (for example, `_variables.scss`), especially when you're working on a larger project. This way, you can easily manage all of your color variables in one place.